demo/src/main/java/com/midtrans/demo/DemoConfigActivity.java

```diff
@@ -1294,25 +1295,10 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean checked) {
     }
 
     private void initPaymentChannelsSelections() {
-        SelectPaymentMethodListViewModel listViewModel = LocalDataHandler.readObject(PAYMENT_CHANNELS_TYPE, SelectPaymentMethodListViewModel.class);
-        if (listViewModel != null
-                && listViewModel.getEnabledPayments() != null
-                && !listViewModel.getEnabledPayments().isEmpty()) {
-            enabledPayments = AppUtils.mapPaymentMethods(this, listViewModel.getEnabledPayments());
-            paymentChannelsTitle.setText(R.string.payment_channels_selected);
-            if (isContainEChannel(enabledPayments)) {
```

Comment:
I think this logic to handle selected enabled payment should not be removed. Because sdk has a feature to only show some several payment channels, not all list.

Reply:
This is in DemoConfig right?? not in the core logic. I think for the demo we can skip this feature